apps/design/design.md

```diff
@@ -140,6 +141,10 @@ Since we have representation from the Go SDK team, we might have them go bridge
 the current implementation using some of that code so its possible for our users to store the
 cert in Keyvault.
 
-## Notes:
+## Logging
 
-Do we need: AcquireTokenSilent()??  Seems like we could just bake this into other acquire calls automatically????
\ No newline at end of file
+For errors, see [error design](../errors/error_design.md).
+
+This library does not log personal identifiable information (PII). For a definition of PII, see https://www.microsoft.com/en-us/trust-center/privacy/customer-data-definitions. MSAL Go does not log any of the 3 data categories listed there. 
```

Comment:
But there are `fmt` calls all throughout the library, no? App developers can redirect stdout to their logs. For all intents and purposes, the `fmt` calls are like logging.

Reply:
There are, however not all `fmt` functions write to stdout. MSAL clients call `fmt.Errorf` and `fmt.Sprintf`, which format strings without printing them.